src/SimpleRADOSStriper.cc

```diff
@@ -46,7 +46,7 @@
 
 using ceph::bufferlist;
 
-#define dout_subsys ceph_subsys_client
+#define dout_subsys ceph_subsys_cephsqlite
```

Comment:
since the only user of the minimal version of rados striper is cephsqlite, the change looks good to me.